datafusion/core/src/datasource/listing/mod.rs

```diff
@@ -67,6 +67,8 @@ pub struct PartitionedFile {
     pub partition_values: Vec<ScalarValue>,
     /// An optional file range for a more fine-grained parallel execution
     pub range: Option<FileRange>,
+    /// Optional statistics that describe the data in this file
```

Comment:
I think we should also provide some more guidance here for example

```suggestion
    /// Optional statistics that describe the data in this file if known.
    ///
    /// DataFusion relies on these statistics for planning so if they are incorrect
    /// incorrect answers may result.
```